src/typings/prop-types.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'prop-types' {
+    import React = require('react');
+
+    export const bool: {
+        isRequired: React.Validator<any>
```

Comment:
This is probably insufficient and will strictly work only for `PropTypes.bool`, any other prop type will fail